server/src/main/java/org/elasticsearch/indices/SystemIndexDescriptor.java

```diff
@@ -225,7 +225,7 @@ protected SystemIndexDescriptor(
         int indexFormat,
         String mappingsNodeVersionMetaKey,
         String origin,
-        Version minimumNodeVersion,
+        @Deprecated Version minimumNodeVersion,
```

Comment:
Is this still here because it's used by serverless?

Reply:
No, it's just because the x-pack Security plugin still uses the minimumNodeVersion for a little custom index management. The security team has a ticket for removing it.